src/Configuration/ECSConfigBuilder.php

```diff
@@ -128,6 +130,8 @@ public function __invoke(ECSConfig $ecsConfig): void
                 $ecsConfig->disableParallel();
             }
         }
+
+        $ecsConfig->reportingRealPath($this->reportingRealPath === null ? false : $this->reportingRealPath);
```

Comment:
this will flip-flop false on multi config as new ECSConfigBuilder recreated on next import files when `configure()` is called, instead, set if null:

```php
if ($this->reportingRealPath !== null) {
   $ecsConfig->reportingRealPath($this->reportingRealPath);
}
```

Reply:
I've tried this approach first. But there is a use case that did not be respected. Keep default value to `false` set, if you don't specify `->withRealPathReporting()` on ecs.php config file.

Reply:
fluent configuration can be defined in multiple imports, in every import, `ECSConfigBuilder` re-created, that's why define compare `!== null` is needed, otherwise, when we define in one config `true`, and other config `null`, it became `false` again as re-setted.

see issue why parallel re-setted

- https://github.com/easy-coding-standard/easy-coding-standard/issues/180
- https://github.com/easy-coding-standard/easy-coding-standard/pull/181

Reply:
OK I heard, but if I follow this implementation, and we don't specify `->withRealPathReporting()` on ecs, then we got an issue 

```
Warning: Undefined array key "reporting-realpath" in /shared/backups/forks/easy-coding-standard/src/DependencyInjection/SimpleParameterProvider.php on line 59

Fatal error: Uncaught TypeError: Symplify\EasyCodingStandard\DependencyInjection\SimpleParameterProvider::getBoolParameter(): Return value must be of type bool, null returned in /shared/backups/forks/easy-coding-standard/src/DependencyInjection/SimpleParameterProvider.php:59
```

Reply:
You need to early set in https://github.com/easy-coding-standard/easy-coding-standard/blob/main/config/config.php 

```diff
return ECSConfig::configure()
    ->withParallel()
    ->withSpacing(indentation: Option::INDENTATION_SPACES, lineEnding: PHP_EOL)
    ->withCache(directory: $cacheDirectory, namespace: $cacheNamespace)
    ->withFileExtensions(['php'])
    ->withSkip([])
    ->withPaths([])
+   ->withRealPathReporting(false);
```

Reply:
Perharps we should think to have a default value if parameter does not exists on these methods : 

```
    public static function getStringParameter(string $key): string
    {
        return self::$parameters[$key];
    }

    public static function getIntParameter(string $key): int
    {
        return self::$parameters[$key];
    }

    public static function getBoolParameter(string $key): bool
    {
        return self::$parameters[$key];
    }
```
Something like 
```
    public static function getBoolParameter(string $key, $default = false): bool
    {
        return self::$parameters[$key] ?? $default;
    }
```

Reply:
I think that needs different PR as changing API signature.

Reply:
> You need to early set in https://github.com/easy-coding-standard/easy-coding-standard/blob/main/config/config.php

I don't known that there is this possibility ! Thanks to point it to me

Reply:
Resolved by commit 4e19addfa0598b7db0949bec6a5a6c9da2b5f282